FordLabs/PeopleMover
# Backend security

## Issue
Connects #98 

## What was done
- [x] Secured back-end

## How to test
1. Make sure that all the application works. It affects every part of the application
2. use postman without the bearer token to access end points and see if you get rejected. If you need a list of all the end points: GOOD LUCK!!! Ask enginners to try a few of them
